rancher/rancher-services/storage-service/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Storage Service
+layout: rancher-default
+
+---
+
+## Storage Service
+
+_Available as of v0.47.0+_
+
+As part of the [Rancher catalog]({{site.baseurl}}/rancher/rancher-ui/applications/catalog/), Rancher provides a storage service. Currently, the storage services are pre-fixed with `Convoy` in the catalog. 
```

Comment:
My two cents:

(I presume that Rancher catalog is a new feature)

Rancher catalog provides storage services that are capable of exposing volumes to containers. You can access the storage services.....
(I do not have the latest build, so it's not clear to me what's meant by pre-fixed with Convoy)